core/controllers/base.py

```diff
@@ -410,18 +417,20 @@ def _render_exception_json_or_html(self, return_type, values):
             values: dict. The key-value pairs to include in the response.
         """
 
-        if return_type == feconf.HANDLER_TYPE_JSON:
-            self.render_json(values)
-        elif return_type == feconf.HANDLER_TYPE_HTML:
+        method = self.request.environ['REQUEST_METHOD']
+
+        if(return_type == feconf.HANDLER_TYPE_HTML and
```

Comment:
Missing space before '('

@apb7 -- shouldn't the linter catch this?

Reply:
done